scripts/runhelpers.cake

```diff
@@ -134,13 +135,21 @@ ExitStatus Run(string command, string arguments, RunOptions runOptions)
     Context.Log.Write(Verbosity.Diagnostic, LogLevel.Debug, "  Arguments: {0}", arguments);
     Context.Log.Write(Verbosity.Diagnostic, LogLevel.Debug, "  CWD: {0}", workingDirectory);
 
-    var process = System.Diagnostics.Process.Start(
-            new ProcessStartInfo(command, arguments)
-            {
-                WorkingDirectory = workingDirectory,
-                UseShellExecute = false,
-                RedirectStandardOutput = runOptions.Output != null
-            });
+    var pInfo = new ProcessStartInfo(command, arguments)
```

Comment:
What's this? Hungarian notation for a pointer to some `Info` class? :smile:

How about `startInfo`?